vm-virtio/src/console.rs

```diff
@@ -618,30 +621,26 @@ impl VirtioDevice for Console {
 }
 
 virtio_pausable!(Console);
-const CONSOLE_SNAPSHOT_ID: &str = "virtio-console";
 impl Snapshottable for Console {
     fn id(&self) -> String {
-        CONSOLE_SNAPSHOT_ID.to_string()
+        self.id.clone()
```

Comment:
You could do `String::new("_console0")`

Reply:
I really don't want the id to be chosen by the device.